docs/references/strategies/languages/python/setuptools.md

```diff
@@ -58,7 +94,7 @@ true `install_requires` list from the CLI's view. For example, FOSSA CLI does no
 
 ## Examples
```

Comment:
[Optional] Let's add this in FAQ explicitly (I found that this helps a lot in triaging/confirming behaviour for on-call and sales/support team)

## F.A.Q

### Why is `fossa-cli` reporting incorrect transitive dependencies or edges?

`fossa-cli` uses `python -m pip list` and `python -m pip show` command to infer
transitive dependencies, and it's edges. It's paramount that you are in project's [venv](https://docs.python.org/3/tutorial/venv.html)
prior to invoking `fossa analyze` command. If you are not in project's virtual environment, 
`fossa-cli` will use global python environment to infer transitive dependencies, 
edges, and dependency versions.

For example, you should always invoke `fossa-cli` in project's python's virtual environment. 
```bash
python -m venv ~/envs/my-venv
source ~/envs/my-venv/bin/activate # this command may differ depending on your python version!
pip install -r requirements.txt

fossa analyze
```

If you are unable to get any transitive dependencies, ensure you can run following command in project's directory: 
- `python -m pip list --format json` 
- `python -m pip show`